docs/Contributing-to-Tachyon.md

```diff
@@ -142,7 +142,8 @@ following list, and then submit a pull request. You are also welcome to address
 issues in our [JIRA](https://tachyon.atlassian.net/browse/TACHYON).
 Here are a list of
 [tasks](https://tachyon.atlassian.net/issues/?jql=project%20%3D%20TACHYON%20AND%20labels%20%3D%20Beginner%20AND%20status%20%3D%20OPEN)
-for beginners. For a tutorial, see the GitHub guides on
+for beginners. Please limit 2 beginner tasks per new contributor.
```

Comment:
why " Afterwards, try some intermediate tasks, or ask in the [User Mailing List](https://groups.google.com/forum/?fromgroups#!forum/tachyon-users)." is not here?